docs/usage.md

```diff
@@ -348,7 +348,10 @@ will have the following effects:
 Note the following important points and limitations for setting up:
 
 * The TSV must use actual tabs (not spaces) between cells.
+* The input FASTQ filenames are discarded after FastQC, all other downstream results files are based on `Sample_Name`, `Library_ID` and `Lane` columns for filenames.
 * *File* names must be unique regardless of file path, due to risk of over-writing (see: [https://github.com/nextflow-io/nextflow/issues/470](https://github.com/nextflow-io/nextflow/issues/470)).
+  * At different stages of the merging process, (as above) nf-core/eager will use as output filenames the information from the `Sample_Name`, `Library_ID` and `Lane` column columns for filenames.
+  * In other words, your .tsv file must not have rows with `Library1` and `Library1` for both `SampleA` and `SampleB`. While nf-core/eager would not try to _merge_ these, in some stages of the pipeline output files names would be the same, and would overwrite the other if the files are output to the same `results/` subdirectory.
```

Comment:
So you are saying that each `Library_ID` entry has to be unique as well? This sounds all a little complicated to me. But I did not read all other docs, so I might be just confused because of lacking background knowledge.

```suggestion
  * Entries in `Library_ID` must be unique.
  * In other words, your `.tsv` file must not have rows with `Library1` and `Library1` for both `SampleA` and `SampleB`. While nf-core/eager would not try to _merge_ these, in some stages of the pipeline output files names would be the same, and would overwrite the other if the files are output to the same `results/` subdirectory.
```

Reply:
Yes it is complicated because (personal opinion:) people do stupid complicated things in our field and it's hard to get around :sweat_smile:.

A lot of this would be removed with DSL2 as we will develop a proper validator to remove a lot of this stuff